src/main/java/com/linkedin/kmf/services/CommitLatencyMetrics.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2020 LinkedIn Corp. Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.linkedin.kmf.services;
+
+import java.util.Map;
+import org.apache.kafka.common.MetricName;
+import org.apache.kafka.common.metrics.Metrics;
+import org.apache.kafka.common.metrics.Sensor;
+import org.apache.kafka.common.metrics.stats.Avg;
+import org.apache.kafka.common.metrics.stats.Max;
+import org.apache.kafka.common.metrics.stats.Percentile;
+import org.apache.kafka.common.metrics.stats.Percentiles;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * The CommitLatencyMetrics class contains methods that measures and
+ * determines the latency of Kafka consumer offset commit().
+ */
+public class CommitLatencyMetrics {
```

Comment:
Some general suggestion/recommendation for APIs on this class. I would consider to have APIs such as:

```java
class CommitLatencyMetrics {
    public void recordCommitStart();
    public void recordCommitComplete();
}
```

There are two advantages If APIs are designed this way

1. We don't have to expose the actually numerical start timestamp values to users and we don't have to make users to generate the complete time stamp and do the calculation to figure out how many milliseconds have passed since the commit start and etc. All users need to do is to record when commit starts and record again when commit finishes.

2. It would be easier to detect abnormal/incorrect situations since the `recordCommitStart()` and `recordCommitComplete()` are always expected to called alternatively with `recordCommitStart()` being expected to be called first.